spring-media/vussr
# RED-1779: Fix clean dist folder

Task: https://jira.bildtools.de/browse/RED-1779

**Proposed changes:**
- upgrade clean-webpack-plugin to version 2
- provide config path

**How to test:**
- add the branch to red-deliver: `yarn add spring-media/vussr#bugfix/RED-1779-clean-dist-folder`
- build red-delivery using production flag: `NODE_ENV=production yarn build`
- add any test file to the 'dist' folder
- run build with production flag again and check if your test file still exist
